psydac/api/fem.py

```diff
@@ -843,7 +980,6 @@ def allocate_matrices(self):
         self._global_mats = list(global_mats.values())
         return global_mats.values()
```

Comment:
Same comments as for `DiscreteBilinearForm`:

- `allocate_matrices` should not return anything
- `construct_arguments` should not change `self`

Reply:
done in [ec10ae7](https://github.com/pyccel/psydac/pull/213/commits/ec10ae70e2106c6cc339a5aa2ae5ac81c351a7f6)